website/client/src/components/groups/invitationListModal.vue

```diff
@@ -0,0 +1,271 @@
+<template>
+  <b-modal
+    id="invitation-list"
+    size="md"
+    :hide-header="true"
+    :hide-footer="true"
+  >
+    <div class="dialog-close">
+      <close-icon @click="close()" />
+    </div>
+    <h2 class="text-center textCondensed" v-once>
+      {{ $t('invitations') }}
+    </h2>
+    <div
+      v-for="member in members"
+      :key="member._id"
+      class="member-row"
+    >
+      <div class="class-icon">
+        <class-badge
+          v-if="member.stats"
+          :member-class="member.stats.class"
+          style="--badge-size: 40px"
```

Comment:
Given this is a component, we should be able to pass a prop rather than use an inline style